azext_edge/tests/edge/checks/int/helpers.py

```diff
@@ -169,9 +169,11 @@ def assert_general_eval_custom_resources(
     for namespace, kubectl_items in sorted_items.items():
         assert namespace in namespace_dict
         check_names = []
-        for item in namespace_dict[namespace]["evaluations"]:
+        # filter out the kubernetes runtime resource evals using /, only check the CRD evals
```

Comment:
will you check the resource evals with /?

Reply:
yeah, with the non CRD eval we have in check, it will have a prefix of "pod/*" or "service/*"